controllers/numaresourcesoperator_controller_test.go

```diff
@@ -56,18 +56,19 @@ import (
 )
 
 const (
-	testImageSpec = "quay.io/openshift-kni/numaresources-operator:ci-test"
+	testImageSpec     = "quay.io/openshift-kni/numaresources-operator:ci-test"
+	defaultOCPVersion = "v4.11"
```

Comment:
I'm trying to think if using `v4.10` would make any difference. I guess we can keep 4.11.

Reply:
Yeah, it does not make a significant difference the version we use. I had to provide a value and thought its best to use the latest version here.

Reply:
that's fine indeed!